### ISSUE TITLE
Dear authors, are the leraning rates for Full FT same as S2FT (reported in Table 6)?

### ISSUE THREAD

**Issue by yuang-lee** [Infini-AI-Lab/S2FT#3]
I am wondering if the learning rates of S2FT under only 1% trainable parameters setting required a larger learning rate than that of Full finetune(100%)?

Thank you!
